src/Managers/DataManager.py

```diff
@@ -33,13 +31,15 @@ def upload_file(self, file_chooser: Any, output_handler: Any) -> None:
                 print("Please, select the file first!\u274C")
             return
 
-        self._data.file = np.loadtxt(file_path, skiprows=1)
-        self._data.headers = list(
-            pd.read_csv(file_path, nrows=1, header=0, sep="[ ]{1,}", engine="python")
-        )
+        if file_path.split(".")[-1] == 'txt':
+           self._data.file = pd.read_csv(file_path, sep = " ")
+        elif file_path.split(".")[-1] == 'csv':
+           self._data.file = pd.read_csv(file_path)
+        elif file_path.split(".")[-1] == 'tsv':
+           self._data.file = pd.read_csv(file_path, sep = "\t")
 
         with output_handler:
-            print("Your file is successfully uploaded!\u2705")
+            print("Your file is successfully uploaded!\u2705 {0}".format(file_path))
```

Comment:
I don't think we should print the file path itself, because by that point the user will already use it to upload the file to the server itself.